descholar-ceo/AnnounceIT
# #170797154 As a developer implement TTL feedbacks

#### What does this PR ​ do​ ?
As a user, I should be to navigate in the application so easily

#### Description of Task to be completed?
* Fix index page
    * Re-Align the brand name
    * Make the enter button clickable fully
    * Re-structure the explanation and interests of announceIT app

* Fix Signup and Signin page
    * Make background picture fixed
    * Make signup and signin look the same and both centered
    * Change the padding
    * Try input `<input type="email"/>`
    * Change `Eg.:` to `Eg:` in placeholder

* Refactor the css styles
    * Use already defined classes and colors instead of re-defining in element specific styles

* Change the create to create announcement in navbar of a user

* Add more flexibility to search bar (search by announcement name, status, id)

* Make search bar place-holder visible

* Fix create-announcement page

* Re-style the table displaying list of all announcement

* Fix the admin page
   * Add shadows to dashboard div
    * Enhance padding to profile details

* Make a single announcement clickable
  
* Make user view details of an announcement whenever he/she clicks an announcement

#### Remaining tasks
    * Responsiveness
    * Enhancing colors

#### How should this be manually tested?
* Clone this repository in case you don't have it using this command `git clone https://github.com/descholar-ceo/AnnounceIT.git`
* If you use [VSCode](https://code.visualstudio.com/) as code editor, install [_live-server_](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer), and click on _Go Live_

* Click on enter button from homepage, 

* And then you will be redirected in the browser, here you will need to go to the root of the url you have in url-bar, and enter this to navigate as a normal user `http://127.0.0.1:5500/UI/html/index.html`, or `http://127.0.0.1:5500/UI/html/admin-side/admin-page.html` to navigate as an admin

* Or if you don't use [VSCode](https://code.visualstudio.com/) as text editor, after cloning the repo, you navigate in `AnnounceIT/UI/html/` and you will see different files, you choose `index.html` to navigate as a normal user or go to `AnnounceIT/UI/htm/admin-side/` and choose admin-page.html to navigate as an admin and right click and open with your preferred web browser

#### Any background context you want to provide?
* It would be better if you use  [VSCode](https://code.visualstudio.com/)

#### What are the relevant pivotal tracker stories?
[170797154](https://www.pivotaltracker.com/n/projects/2429498#)

#### Screenshots (​ if​ appropriate)
![Screenshot from 2020-01-21 07-22-04](https://user-images.githubusercontent.com/56886064/72777747-62f38280-3c1f-11ea-8b5c-906d9bed0c76.png)
![Screenshot from 2020-01-21 07-22-23](https://user-images.githubusercontent.com/56886064/72777748-62f38280-3c1f-11ea-93cf-2b72cbdf94f3.png)
![Screenshot from 2020-01-21 07-22-53](https://user-images.githubusercontent.com/56886064/72777750-638c1900-3c1f-11ea-944a-cb29b4fc2fb8.png)
![Screenshot from 2020-01-21 07-23-06](https://user-images.githubusercontent.com/56886064/72777751-638c1900-3c1f-11ea-8b8f-ecaf70b76c4d.png)
![Screenshot from 2020-01-21 07-23-32](https://user-images.githubusercontent.com/56886064/72777752-6424af80-3c1f-11ea-883e-42f63119678a.png)
![Screenshot from 2020-01-21 07-23-47](https://user-images.githubusercontent.com/56886064/72777753-6424af80-3c1f-11ea-8ab1-a6215e64b484.png)
![Screenshot from 2020-01-21 07-24-02](https://user-images.githubusercontent.com/56886064/72777754-64bd4600-3c1f-11ea-96ea-94e1970afff7.png)
![Screenshot from 2020-01-21 07-24-16](https://user-images.githubusercontent.com/56886064/72777755-6555dc80-3c1f-11ea-95fe-23988fa9f7f6.png)
![Screenshot from 2020-01-21 07-24-35](https://user-images.githubusercontent.com/56886064/72777756-6555dc80-3c1f-11ea-990d-5fc943f96508.png)
![Screenshot from 2020-01-21 07-24-55](https://user-images.githubusercontent.com/56886064/72777758-65ee7300-3c1f-11ea-9e71-c8e4ef8ac74c.png)
![Screenshot from 2020-01-21 07-25-19](https://user-images.githubusercontent.com/56886064/72777759-65ee7300-3c1f-11ea-842c-53c7071b68c6.png)
![Screenshot from 2020-01-21 07-25-32](https://user-images.githubusercontent.com/56886064/72777760-66870980-3c1f-11ea-99a5-8d0f1763437f.png)
![Screenshot from 2020-01-21 07-25-48](https://user-images.githubusercontent.com/56886064/72777761-671fa000-3c1f-11ea-8d9d-2e3068ea3833.png)



#### Questions:
No Question
